default.hbs

```diff
@@ -33,22 +33,8 @@
     <script defer src="https://cdn.freecodecamp.org/news-assets/algolia/autocomplete-0-36-0/autocomplete.min.js"></script>
 
     {{!-- i18n links, trending links, and Algolia index --}}
-    <script type="text/javascript" src="{{asset "config/en/main.js"}}"></script>
-    <script type="text/javascript" src="{{asset "config/en/footer.js"}}"></script>
-
-    <script type="text/javascript" src="{{asset "config/es/main.js"}}"></script>
-    <script type="text/javascript" src="{{asset "config/es/footer.js"}}"></script>
-
-    <script type="text/javascript" src="{{asset "config/zh-cn/main.js"}}"></script>
-    <script type="text/javascript" src="{{asset "config/zh-cn/footer.js"}}"></script>
-
-    {{!-- Setup global links and index objects for locale --}}
-    <script type="text/javascript" src="{{asset "config/setup-locale.js"}}"></script>
-
-    {{!-- Runkit embeds - NOT WORKING? TODO --}}
-    {{!--
-    <script src="https://embed.runkit.com"></script>
-    --}}
+    <script defer type="text/javascript" src="assets/config/{{@site.lang}}/main.js"></script>
+    <script defer type="text/javascript" src="assets/config/{{@site.lang}}/footer.js"></script>
```

Comment:
Unfortunately, we'll run into caching problems if we do this.  Ghost gives these `max-age=31536000`, but without the `asset` helper it doesn't generate the cache-busting hash.

Reply:
For what it's worth, I'm no longer seeing render-blocking JS being flagged up by Lighthouse.  The main remaining issue is the server response, which 11ty should :crossed_fingers: be able to improve.